### PR TITLE
Debian package

### PR DESCRIPTION
**Pull request by tkluck**
Here's the debian packaging code. There seems to be a little merge conflict with what you've done @kdehairy ; let me look into that.

### CODE REVIEW

**Comment by tkluck** [comment]
Rebased, fixed merge errors, and **force pushed** (mind you! In case you happen to have pulled it already).

File: debian/README.Debian [hunk 1]
```diff
@@ -0,0 +1,12 @@
+pac4cli for Debian
+-----------------
+
+We install the python packages txdbus and systemd at post-install time through
+pip. The former since it hasn't been packaged yet; the latter because we need a
+later version.
```
Comment (kdehairy) [inline debian/README.Debian]: Can you specify which min version we need for systemd?

File: debian/postinst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+install_virtualenv() {
+    echo "Stopping pac4cli service (if it is running)..."
+    systemctl stop pac4cli.service || true
+    virtualenv -p python3 --system-site-packages /opt/pac4cli
+    /opt/pac4cli/bin/pip install txdbus systemd
```
Comment (kdehairy) [inline debian/postinst]: Since we opted to install the rest of dependencies system-wide, why do we install those specifically in a virtualenv?

**Comment by kdehairy** [comment]
Good job!

Good that you specified a license. I was going to ask you about it.

**Pull request merged**